packages/web/components/CardResultsProvider.tsx

```diff
@@ -89,30 +89,36 @@ export default function CardResultsProvider({
   );
   const [hasMoreCards, setHasMoreCards] = useState(true);
 
+  const formatDateForSupabase = (date) => {
+    const isoString = date.toISOString();
+    const formattedDate = isoString.replace("T", " ").slice(0, -1) + "+00";
```

Comment:
Not sure why this specific modification is needed. Smells like there might be a less custom solution. Perhaps something like: https://github.com/orgs/supabase/discussions/2839#discussioncomment-1184927

Reply:
Thanks